courses/service-course/steps/06_clients-masterdata/en.md

```diff
@@ -204,6 +204,10 @@ Now you are good to go!
    --header 'Content-Type: application/json'
    ```
 
+    > **Attention**: If you have a version of Windows earlier than Windows 10, version 1803, it’s necessary to [download and install _cURL_](https://curl.se/windows/). Otherwise, it is installed by default.
+    >
+    > To run the command _cURL_ on **Windows**, it's necessary to substitute single quotes (`'`) for double quotes (`"`).
```

Comment:
cURL aqui é o comando em si ou o tipo do comando? Se for o comando, recomendo trecho de código (``), se for cURL no sentido amplo recomendo negrito (****).